CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+ * `TimelockController`: During deployment, the TimelockController used to grant the `TIMELOCK_ADMIN_ROLE` to the `admin` parameter, it will now assign the `DEFAULT_ADMIN_ROLE` to the `admin` parameter to be consistent with the rest of the repository.([#3799](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3799))
```

Comment:
This changelog entry is not very accurate. It's not just about the `admin` parameter, the timelock itself also has the admin role.

We should say something about how the contract used to use a special admin role and it will now use the default like all other contracts.